app/datarequest/datamanagerreview.js

```diff
@@ -4,51 +4,67 @@ import { render } from "react-dom";
 import Form from "react-jsonschema-form";
 import DataSelection, { DataSelectionCart } from "./DataSelection.js";
 
-var datarequestSchema   = {};
-var datarequestUiSchema = {};
-var datarequestFormData = {};
-var prSchema   = {};
-var prUiSchema = {};
-var prFormData = {};
-
-// Get the schema, uiSchema and formData of the data request to be reviewed by the data manager.
-// Then render the data as a disabled form
-axios.all([
-    axios.get("/datarequest/datarequest/schema"),
-    axios.get("/datarequest/datarequest/data/" + requestId),
-    axios.get("/datarequest/datarequest/preliminaryReviewSchema"),
-    axios.get("/datarequest/datarequest/preliminaryReviewData/" + requestId)
-    ])
-    .then(axios.spread((schemaresponse, dataresponse,
-                        prschemaresponse, prdataresponse) => {
-        datarequestFormData = dataresponse.data;
-        datarequestSchema   = schemaresponse.data.schema;
-        datarequestUiSchema = schemaresponse.data.uiSchema;
-        prFormData          = prdataresponse.data;
-        prSchema            = prschemaresponse.data.schema;
-        prUiSchema          = prschemaresponse.data.uiSchema;
-
+$(document).ready(function() {
```

Comment:
Same change as in app/datarequest/preliminaryreview.js:

` document.addEventListener("DOMContentLoaded", async function() { `